### DISCUSSION TITLE
Share your accessibility pet peeves

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5498] [Accessibility]
In various stages of production — from design to deployment — we have all encountered accessibility problems. In your role, which pet peeves do you see the most? 

I’ll start:
One of my pet peeves is primary headers styled using a distinct font or bold color but not designated with an H1 in the code or stylesheet.

Also I commonly see headers out of order on a page, which causes frustration, especially for screen reader users who depend on structured content to easily scan pages.

Share your experiences with us and together we can work on fixes both inside and outside our community!

**Comment by vickinssl** [comment]
OMG, YES, people not using headers in a meaningful, semantic way. I don't expect my scientist writers to pay attention, but I really wish my actual comms people cared. The WordPress block editor makes it easy to add visual styles to the correct headers within a page, so it's not just "I want bigger fonts!" We can have those AND be accessible!

My pet peeve: user reluctance to make PDFs accessible. It's not that hard! There are a million tutorials on the web (including from digital.gov)! And yet! It's like pulling teeth.

I have two years and change until I retire. I've been managing this website for 25+ years. Me, when I'm out: popcorn.gif

**Reply by amycole501** [reply]
I agree about PDFs and how often they're in accessible. Some people opt to delete a PDF instead of fixing it which doesn't address the underlying issue that the file still won't be accessible just because you delete it. What if the file gets shared some other way?

Depending on the content, some PDF fixes can be done in a matter of minutes! I like the training offered on [Section 508](https://www.section508.gov/create/documents/), complete with a video tutorial for creating accessible documents; and the [PDF testing checklist](https://www.section508.gov/create/pdfs/authoring-guides/) is very handy. What I love about accessibility checklists is they walk you through a series of prompts and the process becomes a learning opportunity for people who may be new to accessibility.

And great job on over 25 years working on the same website! I'm sure you've seen a lot of changes during that time :)

**Reply by vickinssl** [reply]
Yep, I've seen it all come and go. When I started, people were using the visual builder in Netscape, and Microsoft Front Page. (*shudder*) We've come a long way!

**Comment by mgifford** [comment]
I'd say my pet peeves are PDFs. They are too easy to produce, too hard to make accessible, and very difficult to determine how accessible or inaccessible they are with a quick review. 

Organizations have done lots of work to make their existing PDFs WCAG 2.0 compliant. Will they all need to be revised for 2.1 & 2.2 compliance? 

Much, much better for everyone to stick to HTML until absolutely necessary: 
https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/

**Reply by amycole501** [reply]
Removing the requirement for government reports to be submitted via PDF would help. Most of the recent PDFs I've come across are mandatory reports which include accessibility issues with charts, graphs, tables (with merged cells), and all kinds of various headings that aren't actually headings. 

Also, bridging the gap between print and web digital media has been a challenge especially for agencies who may not have a CMS for their website.  While most people feel comfortable creating a document they may not feel comfortable (nor have permissions) to make a webpage. Lots of internal process changes need to take place before PDFs take more of a back seat to HTML pages. This all circles back to the original issue of making content more inclusive. PDFs are rarely the answer.
